### DISCUSSION TITLE
Community Support Links problem

### DISCUSSION THREAD

**Discussion by pietru2004** [django-cms/django-cms#7667] [Q&A]
Hello I want to ask could somebody refresh the Discord link at https://www.django-cms.org/en/support/
it Says that link is invalid and I wanted to get help with my test deployment.

My Problem:
when I disable debug server gets 500

**Comment by marksweb** [comment]
@pietru2004 You'd be best with the slack link from there. The discord server was a bit of an experiment and the majority of the community are on slack. We've a `#support` channel on there where you can post, but you'll need to do a bit more debugging to establish what the actual error is before anybody can help with that.

**Reply by pietru2004** [reply]
well I have no slack and I already have too much services so if possible I would prefer to limit accounts I create...

**Reply by pietru2004** [reply]
also I was able to fix half of problem - I have coppied static files to folder... I think they for some reason did not appear/ be pulled from docker img

**Reply by pietru2004** [reply]
now the images don't want to load

**Reply by marksweb** [reply]
It's a very basic example, but typically you'd run `collectstatic` in your dockerfile, as you can see from our quick-start project;

https://github.com/django-cms/django-cms-quickstart/blob/main/Dockerfile

**Reply by pietru2004** [reply]
that is how dockerfile on my side looks...
